src/views/Proposal.vue

```diff
@@ -136,7 +136,7 @@
                 target="_blank"
                 class="float-right"
               >
-                {{ $n(payload.snapshot) }}
+                {{ payload.snapshot }}
```

Comment:
If $n isn't working anymore you can use `_numeral` mixin method with the format `0,0`

Reply:
Ok great, in this case we shouldn't use any formatter because it's the block number of the snapshot, and the block number is displayed without formatting throughout the site.

Reply:
It should be displayed with formating like this:
23,456,432
Instead of this
23456432

Reply:
This is the way its show currently